pontoon/insights/utils.py

```diff
@@ -54,17 +54,28 @@ def get_insights(query_filters=None):
     )
 
     latest = snapshots.latest("created_at") if snapshots else None
-    active_users = latest.active_users_last_12_months if latest else None
+    default = active_users_default()
 
     return {
         "dates": [convert_to_unix_time(month) for month in months],
         # Active users
-        "total_managers": latest.total_managers if latest else 0,
-        "total_reviewers": latest.total_reviewers if latest else 0,
-        "total_contributors": latest.total_contributors if latest else 0,
-        "active_managers": active_users["managers"] if active_users else 0,
-        "active_reviewers": active_users["reviewers"] if active_users else 0,
-        "active_contributors": active_users["contributors"] if active_users else 0,
+        "total": {
+            "managers": latest.total_managers if latest else 0,
+            "reviewers": latest.total_reviewers if latest else 0,
+            "contributors": latest.total_contributors if latest else 0,
+        },
+        "active_users_last_month": latest.active_users_last_month
```

Comment:
[nitpick] `black` formats this part in a weird way. Also, Subjectively, I see code repetition in the conditions like `if latest`. Maybe it would be better to move the initialization of the default values outside of this dict, and create a separate block? e.g.
```
if latest:
   results['total'] = {}
else:
   results['total'] = {}
```